forum/users/api_view.py

```diff
@@ -2,12 +2,21 @@
 from rest_framework.permissions import IsAuthenticated, AllowAny
 from rest_framework_simplejwt.authentication import JWTAuthentication
 from rest_framework.response import Response
-from rest_framework_simplejwt.tokens import RefreshToken
+from rest_framework_simplejwt.tokens import RefreshToken, AccessToken, TokenError
 from .models import User
 from .serializers import UserSerializer
 from .permissions import IsAdmin, IsOwner
 from rest_framework.throttling import AnonRateThrottle
+from rest_framework.views import APIView
+from django.contrib.auth import get_user_model
+from rest_framework_simplejwt.exceptions import AuthenticationFailed
+from django.core.exceptions import ObjectDoesNotExist
+from rest_framework.throttling import AnonRateThrottle
+from users.models import User
+from rest_framework_simplejwt.tokens import RefreshToken
+
```

Comment:
There are redundant and repeated imports like AnonRateThrottle, which is imported twice, and `User` which is already imported using `get_user_model()`